docs/conditions.md

```diff
@@ -0,0 +1,52 @@
+# Conditions
+
+This document defines `Conditions` and their capabilities.
+
+*NOTE*: This feature is currently a WIP.
+
+---
+
+- [Syntax](#syntax)
+  - [Check](#check)
+  - [Parameters](#parameters)
+
+- [Examples](#examples)
+
+## Syntax
+
+To define a configuration file for a `Condition` resource, you can specify the
+following fields:
+
+- Required:
+  - [`apiVersion`][kubernetes-overview] - Specifies the API version, for example
+    `tekton.dev/v1alpha1`.
+  - [`kind`][kubernetes-overview] - Specify the `Condition` resource object.
+  - [`metadata`][kubernetes-overview] - Specifies data to uniquely identify the
+    `Condition` resource object, for example a `name`.
+  - [`spec`][kubernetes-overview] - Specifies the configuration information for
+    your `Condition` resource object. In order for a `Condition` to do anything,
+    the spec must include:
+    - [`check`](#check) - Specifies a container that you want to run for evaluating the condition 
+
+[kubernetes-overview]:
+  https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/#required-fields
+
+### Check
+
+The `check` field is required. You define a single check to define the body of a `Condition`. The 
+check must specify a container image that adheres to the container contract. The container image 
```

Comment:
"container contract" could link to https://github.com/tektoncd/pipeline/blob/master/docs/container-contract.md

Reply:
done